README.md

```diff
@@ -1 +1,23 @@
-# clms-stac
\ No newline at end of file
+# clms-stac
+This repository contains example STAC collections and items of CLMS **Corine Land Cover** and **Vegetation Phenology and Productivity** products with **product schemas** for validation and maintanence.
+
+```
+.
+├── stacs
+│   ├── corine-land-cover
+│   │   ├── clc2018_clc2018_v2018_20_raster100m
+│   │   │   └── clc2018_clc2018_v2018_20_raster100m.json (STAC item)
```

Comment:
General comment 

Are there any best practices for naming files ?
can we name the files to indicate what they are e.g 

```
clc2018_clc2018_v2018_20_raster100m-item.json
corine-land-cover-raster-collection.json
```

same to vpp

Reply:
Usually we name it as the source file (see [example](https://github.com/stactools-packages/sentinel2/tree/main/examples/sentinel2-l2a-example/S2A_MSIL2A_20190212T192651_R013_T07HFE_20201007T160857)). But I think it won't be an issue if we want to add `item` or `collection` to the file name.